### PR TITLE
Add support for Python 3

### PR DESCRIPTION
**Pull request by asgeroverby**
Minimal support for Python 3.

All tests pass with:

Robot Framework 3.0 (Python 2.7.5 on win32)
Robot Framework 3.0 (Python 3.4.3 on win32)

Note: I think the notice on Python 3 in QuickStart.rst should be updated if this pull request is accepted.

### CODE REVIEW

File: sut/login.py [hunk 1]
```diff
@@ -47,14 +47,14 @@ def change_password(self, username, old_pwd, new_pwd):
...
-        except ValueError, err:
+        except ValueError as err:
             return 'Changing password failed: %s' % err
         else:
             return 'SUCCESS'
 
     def save(self):
         with open(self.db_file, 'w') as file:
-            for user in self.users.values():
+            for user in list(self.users.values()):
```
Comment (pekkaklarck) [inline sut/login.py]: Is `list` really needed here?

File: sut/login.py [hunk 2]
```diff
@@ -104,22 +104,22 @@ def _validate_password_chars(self, password):
...
     with UserDataBase() as db:
-        print db.change_password(username, old_pwd, new_pwd)
+        print(db.change_password(username, old_pwd, new_pwd))
 
 
 def help():
-    print ('Usage: %s { create | login | change-password | help }'
-           % os.path.basename(sys.argv[0]))
+    print(('Usage: %s { create | login | change-password | help }'
+           % os.path.basename(sys.argv[0])))
```
Comment (pekkaklarck) [inline sut/login.py]: Doesn't this add extra parenthesis?

**Comment by pekkaklarck** [comment]
Good idea! Please look at the notes I added, tough. It would probably be a good idea to add `from __future__ import print_function` to files using `print` too.

It's a good idea also to mention in that the guide supports Python 3. Should it be mentioned both in README.rst and in QuickStartGuide.rst? Could you add such notes too?

**Comment by asgeroverby** [comment]
Your suggestions have been implemented. Notes on Python 3 have been added / updated too.

**Comment by pekkaklarck** [comment]
Great! In a bigger project (e.g. Robot Framework) I would still ask you to squash commits, but I don't think that's really needed here. I also noticed on small typo but can easily fix it myself.

**Comment by pekkaklarck** [comment]
Thanks again for the PR. Decided to create a new release with it: https://github.com/robotframework/QuickStartGuide/releases/tag/1.1

**Pull request merged**